### PR TITLE
Ft reset password #164596175

### PR DESCRIPTION
**Pull request by bekeplar**
## What this PR does?
- Adds user signup UI to the application
- Adds user signin feature to the app
- Adds a user reset password page to the app.

## Stories attached to this PR
[Delivers- #164596083, #164596144, #164596175]
of pivotal tracker

kindly review and advise as necessary dears
@Ivankivu, @phionanamugah, @jenny

### CODE REVIEW

File: UI/templates/signup.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
+<DOCTYPE <!DOCTYPE html>
```
Comment (alexxsanya) [inline UI/templates/signup.html:1]: - Make this line to be simply `<!DOCTYPE html>`

File: UI/templates/reset.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
+<DOCTYPE <!DOCTYPE html>
```
Comment (alexxsanya) [inline UI/templates/reset.html:1]: This line should be simply `<!DOCTYPE html>`

File: UI/static/main.css [hunk 3]
```diff
@@ -0,0 +1,82 @@
+body{
+    margin: 0px;
```
Comment (alexxsanya) [inline UI/static/main.css:2]: I would advise you make you set `margin:auto` for better body balance horizontally

**Comment by bekeplar** [comment]
Thanks dear. will be done.

**Comment by Ivankivu** [comment]
@bekeplar have u made the necessary corrections??

**Comment by bekeplar** [comment]
Working on it

**Pull request merged**